modules/gke/variables.tf

```diff
@@ -59,7 +59,7 @@ variable "initial_node_count" {
 
 variable "machine_type" {
   type        = string
-  default     = "e2-highmem-8"
+  default     = "e2-highmem-16"
```

Comment:
@gtoonstra just realized now, would it be alright to update this here (assuming is being used for other deployments as well where we may not want this type). 

perhaps I should be trying to override this somehow from cloud-infra and leave this as `e2-highmem-8` for non-cityb?

Reply:
We should not override here. By default we sort of "sized" our deployments ok for now. So we should not chagne the defaults.

Having said that, if it were to happen, we'll see changes in our general deployments that we always should inspect and the parameters that change. So even if we merge this, we'd see the changes in other deployments occurring, telling us that something may be off. And then we investigate.

Reply:
So let's revert this. WE should configure main.tf in cloud-infra for specific cases and leave this as a good default.